src/main/java/uk/gov/hmcts/reform/pip/data/management/models/court/Court.java

```diff
@@ -0,0 +1,73 @@
+package uk.gov.hmcts.reform.pip.data.management.models.court;
+
+import com.vladmihalcea.hibernate.type.array.ListArrayType;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import org.apache.commons.text.WordUtils;
+import org.hibernate.annotations.Type;
+import org.hibernate.annotations.TypeDef;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.UUID;
+import java.util.stream.Collectors;
+import javax.persistence.CascadeType;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.OneToMany;
+import javax.persistence.Table;
+
+/**
+ * This class captures the Court Data, which will be persisted in the database.
+ */
+@Entity
+@Data
+@NoArgsConstructor
+@Table(name = "court")
+@TypeDef(
+    name = "list-array",
+    typeClass = ListArrayType.class
+)
+public class Court {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    @Column(columnDefinition = "uuid", insertable = false, updatable = false, nullable = false)
+    @Type(type = "org.hibernate.type.PostgresUUIDType")
+    private UUID courtId;
```

Comment:
Can you please check it to Integer? Frontend is using courtId as INT in models and other places as well.

Reply:
Updated :) Had a discussion, and UUID was a bit excessive especially when the URL's are going to display the UUID.